src/utils/include/com/amazonaws/kinesis/video/utils/Include.h

```diff
@@ -1624,19 +1637,6 @@ typedef struct __ExponentialBackoffRetryStrategyConfig {
 } ExponentialBackoffRetryStrategyConfig;
 typedef ExponentialBackoffRetryStrategyConfig* PExponentialBackoffRetryStrategyConfig;
```

Comment:
I know its not from your PR but could you please fix this too to be consistent with our other typedefs